solutions/LC matrix-block-sum.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: lc-MatrixBlockSum
+source: LeetCode
+title: Matrix Block Sum
+author: Jesse Choe
+---
+
+**Time Complexity:** $\mathcal O(NM)$
+
+<LanguageSection>
+
+<CPPSection>
+
+## Implementation
+
+```cpp
+class Solution {
+    public:
+            vector<vector<int>> matrixBlockSum(vector<vector<int>>& mat, int K) {
+                int n = mat.size(), m = mat[0].size();
+                int arr[n][m];
+                for(int i=0; i<n; i++){
+                    for(int j=0; j<m; j++){
+                        arr[i][j] = mat[i][j];
+                    }
+                }
+                int dp[n+1][m+1];
+                for(int i=0; i<=n; i++){
+                    for(int j=0; j<=m; j++){
+                        dp[i][j] = 0;
+                    }
+                }
+                for(int i=1; i<=n; i++){
+                    for(int j=1; j<=m; j++){
+                        dp[i][j] =  dp[i-1][j]  +  dp[i][j-1]  -  dp[i-1][j-1]  +  arr[i-1][j-1];
```

Comment:
the spacing is still inconsistent (and now there are two spaces???) let me know if it's still unclear what you need to fix.

Reply:
Still unclear lol. Can you suggest change??

Reply:
```suggestion
                        dp[i][j] = dp[i - 1][j] + dp[i][j - 1] - dp[i - 1][j - 1] + arr[i - 1][j - 1];
```
do this for everything else, make sure each operator (`+=-/`) is spaced with exactly one space on each side.

Reply:
```suggestion
						dp[i][j] = dp[i-1][j] + dp[i][j-1] - dp[i-1][j-1] + arr[i-1][j-1];
```

Reply:
Didn't see you alr suggested.